<launch>

  <arg name="topic_to"                    default=""/>
  <arg name="topic_from"                  default=""/>
  <arg name="tf_rebroadcaster_config"     default="$(find interbotix_tf_tools)/config/tf_rebroadcaster.yaml"/>

  <node
    name="tf_rebroadcaster"
    pkg="interbotix_tf_tools"
    type="tf_rebroadcaster"
    output="screen">
    <param name="topic_to"                value="$(arg topic_to)"/>
    <param name="topic_from"              value="$(arg topic_from)"/>
    <param name="filepath_config"         value="$(arg tf_rebroadcaster_config)"/>
  </node>

</launch>
